purchase_ext_sst/views/purchase_order_views.xml

```diff
@@ -97,8 +99,8 @@
         <field name="inherit_id" ref="purchase.view_purchase_order_filter"/>
         <field name="arch" type="xml">
             <xpath expr="//field[@name='name']" position="before">
-                <field name="phone"
-                       filter_domain="['|', ('phone', 'ilike', self), ('mobile', 'ilike', self)]"/>
+                <field name="supplier_phone"
+                       filter_domain="['|', ('supplier_phone', 'ilike', self), ('supplier_mobile', 'ilike', self)]"/>
```

Comment:
I think the current form view is not easy for users to follow at the moment. Can we reorganize the field positions and names. My suggestion is as in below image. We still need to test more to see if the layout is suited for user's operation.
![image](https://user-images.githubusercontent.com/7766939/37351664-89aa8d8a-2716-11e8-8587-9faeee547e25.png)